qiskit/ignis/verification/entanglement/linear.py

```diff
@@ -0,0 +1,155 @@
+'''
+The module linear.py provides the linear
+preparation analogous of parallelize.py.
+'''
+
+from qiskit import *
+from qiskit.circuit import Parameter
+
+
+def get_measurement_circ(n, extent='full'):
+    '''
+    Creates a measurement circuit that can toggle between
+    measuring the first control qubit or measuring all qubits.
+    The default is measurement of all qubits.
+    Args:
+       n: number of qubits
+       extent ('full', 'one'):
+        Whether to append full measurement, or only on the first qubit.
+    Returns:
+       The measurement suffix for a circuit
+    '''
+    q = QuantumRegister(n, 'q')
```

Comment:
This is not part of the usecases we have in the Entanglement context. If the method will be ported to a more generic place, indeed this can/should be done. Otherwise it'll be dead untested code